extensions/query-store/src/settings/configureDialog.ts

```diff
@@ -365,29 +412,36 @@ export class ConfigureDialog {
 		};
 	}
 
-	private createTimeIntervalComponent(): azdata.FormComponent {
-		this.timeIntervalOptionsTopResourceReport = [constants.last5MinsLabel, constants.last15MinsLabel, constants.last30MinsLabel, constants.lastHourLabel, constants.last12HoursLabel, constants.lastDayLabel, constants.last2DaysLabel,
-		constants.lastWeekLabel, constants.last2WeeksLabel, constants.lastMonthLabel, constants.last3MonthsLabel, constants.last6MonthsLabel, constants.lastYearLabel, constants.customLabel];
-
-		this.timeIntervalOptionsOverallReport = [constants.lastHourLabel, constants.lastDayLabel, constants.last2DaysLabel, constants.lastWeekLabel, constants.lastMonthLabel, constants.last6MonthsLabel, constants.lastYearLabel, constants.customLabel];
+	private createTimeIntervalComponent(isOverallResourceReport: boolean = false): azdata.FormComponent {
+		let value;
+		if (isOverallResourceReport) {
```

Comment:
we're starting with just these first two reports, but is this check still accurate when the other query store reports are added?

Reply:
@erinstellato-ms , @kisantia Any thoughts on the above question? Basically there are different values that are populated for time intervals in different reports. Does it make sense to have all the reports present the same values or should I stick to the original ?

Reply:
@SakshiS-harma I never noticed that the drop downs had different intervals.  I suspect it's because aggregating the data (for Overall Resource Consumption) for the last 5 minutes is not easy to calculate in some scenarios.  I'm not clear on why the drop downs are different for the third and fourth screenshots, which have the same time (Regressed Queries).

Reply:
Got it. I think it would be best to keep the values as-is.
@kisantia as for your initial comment, for now the changes work. But when there will be more reports coming in, the logic here would need to be updated. I am adding a comment here for now to make sure to update it in future.

Reply:
Actually, updated it in the next iteration here, so that less work is left for future 👍